Bottom: 050c186b822b9c4e76049b5449940c51edf1c8be
Top:    dc91420798f8dba99d3fcdf35072ca7f4bc5ab1b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-26 14:46:55 +0000

Disable a few noisy debug printf


---

diff --git a/src/polygon1.c b/src/polygon1.c
index c5b2d71..1b70a80 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -241,14 +241,10 @@ node_add_single (VNODE * dest, Vector po)
   VNODE *p;
 
 /* XXX: MAY NOT BE CORRECT IF WE NEED TO SEPARATE STRAIGHT AND CURVED SEGMENTS */
-  if (vect_equal (po, EDGE_BACKWARD_VERTEX (dest)->point)) {
-    pcb_fprintf (stderr, "node_add_single() returning EDGE_BACKWARD_VERTEX (dest);\n");
+  if (vect_equal (po, EDGE_BACKWARD_VERTEX (dest)->point))
     return EDGE_BACKWARD_VERTEX (dest);
-  }
-  if (vect_equal (po, EDGE_FORWARD_VERTEX (dest)->point)) {
-    pcb_fprintf (stderr, "node_add_single() returning EDGE_FORWARD_VERTEX (dest);\n");
+  if (vect_equal (po, EDGE_FORWARD_VERTEX (dest)->point))
     return EDGE_FORWARD_VERTEX (dest);
-  }
   p = poly_CreateNode (po);
   if (p == NULL)
     return NULL;
